lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,200 @@
+|===
+
+We can use `filter` to split our dataset into multiple sub-groups, and then fit models to each sample.
+
+[.lesson-instruction]
+Create helper functions to filter at least three other states, then use them to define the Covid tables for each state.
+
+=== Common Misconceptions
+
+It's extremely common for students to think that filtering a table *changes the original table*. This is NOT how it works in Pyret! Instead, the `filter` function always produces a _new_ table, containing only the Rows for which the supplied function evaluates to `true`
+
+=== Synthesize
+- In what other situations would it be useful to filter a dataset?
+- Can you think of other examples where Simpson's Paradox might arise?
+
+
+== Looking for Patterns @duration{15 minutes}
+
+=== Overview
+
+Students explore their newly-filtered `MA-covid` dataset, trying to fit different kinds of models to it. This section makes heavy use of @link{https://www.Desmos.com, Desmos}, which is an excellent tool for open-ended experimentation.
+
+=== Launch
+
+Make a scatter-plot showing the Covid infection rate for Massachusetts. What kind of model do you think would fit this best?
+
+=== Investigate
+
+[.lesson-instruction]
+- Complete the *first* section on @printable-exercise{graphing-models.adoc}, using the first screen in @starter-file{alg2-covid-desmos} (Linear Models).
+
+Linear models capture _straight-line relationships_, where one quantity varies proportionally based on another. In linear models, we expect the response variable to grow by equal amounts over equal intervals in the explanatory variable.
+
+[.lesson-instruction]
+- Using `lr-plot` in Pyret, come up with the best possible linear model, and write your answer at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are linear models a good fit for this data? Why or why not?
+
+@right{@image{images/MA-covid-linear.png, 300}} If we make the line go from the start to the peak, almost all of the points bulge out below out line of best fit. If we make the line fit the bulge, all the points fall above it. We always have either too many points _below_ the line in the middle or _above_ the line at the end. **It's growing too fast to be fit with a linear model that grows at a constant rate!**
+
+[.lesson-instruction]
+- Complete the *second* section on @printable-exercise{graphing-models.adoc}, using the first screen in @starter-file{alg2-covid-desmos} (Quadratic Models).
+
+Quadratic models capture _parabolic relationships_, where one quantity varies based on the square of another. In quadratic models, we expect the response variable to grow by differing amounts over equal intervals in the explanatory variable.
+
+[.lesson-instruction]
+- Write the best quadratic model you can find at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are quadratic models a good fit for this data? Why or why not?
```

Comment:
I think that if the dataset started at Day 75 instead of Day 0 it would be much clearer to students that quadratic model isn't the best...